dc-cudami-server/dc-cudami-server-backend-jdbi/src/test/java/de/digitalcollections/cudami/server/backend/impl/asserts/IdentifiableAssert.java

```diff
@@ -0,0 +1,31 @@
+package de.digitalcollections.cudami.server.backend.impl.asserts;
+
+import de.digitalcollections.model.identifiable.Identifiable;
+import org.assertj.core.api.AbstractAssert;
+
+/**
+ * The IdentifiableAssert compares to identifiables field by field with the exception of
+ * lastModified.
+ */
+public class IdentifiableAssert extends AbstractAssert<IdentifiableAssert, Identifiable> {
+
+  Identifiable actual;
```

Comment:
```suggestion
  private Identifiable actual;
```
Warum package visible?

Reply:
war keine Absicht :-)